[WIP]2016-07-18-july-2016-economical-software-testing.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "July 2016 Meetup - Economical Software Testing"
+description: ""
+tags: [node.js, javascript, software testing]
+---
+{% include JB/setup %}
+
+The Seattle Node.js Meetup is pleased announce our
+[meetup](xxx)
+on **Thursday, July 27th** Economical Software Testing
+
+As always, you can find all things Seattle Node.js-related at the group website:
+[http://seanode.github.io/](http://seanode.github.io/). And you can reach out to
+us via email at [{{site.author.email}}](mailto:{{site.author.email}}) or Twitter
+at [@{{site.author.twitter}}](https://twitter.com/{{ site.author.twitter }}).
+
+See you soon!
+
+## Agenda
+
+* **6:30 - 7:00pm**: Start with snacks and mingling, courtesy of Mixpo.
+
+* **7:00pm - 8:00pm**: Listen and learn about Economical Software Testing, as well
```

Comment:
Replace `Listen and learn about Economical Software Testing` with `Listen and learn about cost / expense analysis in software testing`